Problems with FW 4.15.250

Hello,

since Harmony Hub Firmware 4.15.250 the plugin throws a lot of errors and can not control Harmony activities or even get correct state.

XMPP for all my Hubs is enabled in new App/FW version.

Will there be a fix soon please?

Hello, when I try that command, I get that error:

```
root@Server:~# npm install -g materik/homebridge-harmony-plugin#develop
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/materik/homebridge-harmony-plugin.git
npm ERR!
npm ERR! Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-02-26T12_30_11_719Z-debug.log

```

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "homebridge-harmonyhub-plugin",
     "description": "Logitech HarmonyHub plugin for HomeBridge",
-    "version": "0.5.5",
+    "version": "0.5.6",
     "repository": {
         "type": "git",
         "url": "git://github.com/materik/homebridge-harmonyhub-plugin.git"
@@ -34,7 +34,7 @@
         "url": "http://materikab.com"
     },
     "dependencies": {
-        "@harmonyhub/client-ws": "1.0.2",
+        "@harmonyhub/client-ws": "1.0.4",
         "@harmonyhub/discover": "1.0.5",
         "lodash": "4.17.11",
         "promise": "8.0.2"

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
     "dependencies": {
         "@harmonyhub/client-ws": "1.0.2",
         "@harmonyhub/discover": "1.0.5",
-        "lodash": "4.17.5",
+        "lodash": "4.17.11",
         "promise": "8.0.2"
     }
 }

